templates/vue-demo-store/components/FrontendNavigationPage.vue

```diff
@@ -30,11 +29,8 @@ if (!categoryResponse.value) {
   throw error.value;
 }
 
-const breadcrumbs = getCategoryBreadcrumbs(categoryResponse.value, {
-  startIndex: 2,
-});
-
-useBreadcrumbs(breadcrumbs);
+const { buildDynamicBreadcrumbs } = useBreadcrumbs();
+buildDynamicBreadcrumbs(props.navigationId);
```

Comment:
it's happening asynchronously, so I have two questions:
1. how are you assuring it's loaded on Server Side Render?
2. isn't there a blink on client side? What about showing previous breadcrumbs and replace them when new ones are loaded? Does it make sense?